### PR TITLE
chore: update grammar

### PR DESCRIPTION
**Pull request by tenthirtyam**
Updates grammar in comments and documentation, as needed.

### CODE REVIEW

**Review by lbajolet-hashicorp — APPROVED** [review]
LGTM! I left a couple nits on some comments, but overall we are good to go.
I'll let you address those and come back for a second review.

Pre-approving for later!

File: builder/vmware/common/driver_parser.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ func parseDhcpConfig(in chan string) (tkGroup, error) {
 
 		case ";":
 			// If we encounter a line-terminator, then the list of tokens we've been
-			// aggregating are ready to be parsed. Afterwards, we can write them
+			// aggregating are ready to be parsed. Afterward, we can write them
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply builder/vmware/common/driver_parser.go:255]: In British English, "afterwards" is the preferred spelling, while "afterward" is more commonly used in American English; both words mean the same thing. 😄

File: builder/vmware/common/driver_parser.go [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ func ReadDhcpConfiguration(fd *os.File) (DhcpConfiguration, error) {
 	}
 
 	// This closure is just to the goro that follows it in recursively walking
```
[the remark this answers is not in the document]
Comment (tenthirtyam) [reply builder/vmware/common/driver_parser.go:1140]: Yeah, that makes no sense. I'll fix that one. Maybe it meant 'go routine'?

[the remark this answers is not in the document]
Comment (tenthirtyam) [reply builder/vmware/common/driver_parser.go]: Resolved in 41d3145. Updated some occurrences ot "goro" to "goroutine" for clarity.

[the remark this answers is not in the document]
Comment (lbajolet-hashicorp) [reply builder/vmware/common/driver_parser.go]: Perfect thanks @tenthirtyam ! Merging now

**Pull request merged**